code-challenges.test.js

```diff
@@ -49,12 +139,54 @@ const studyMinutesWeek2 = {
 }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
+
+
+
+// // Create a function called objectSorter that takes in an object and returns an array of the values sorted from least to greatest
+// // Create a parameter called 'data' as a placeholder for the object we will sort. I used 'data' instead of 'object' since the method we will be using already contains the word 'object'. This will prevent any confusion
+// // Create a variable called 'sorted' to hold the logic
+// // Use 'Object.value()' to extract the values from the keys in an object
+// // Chain .sort() at the end to sort the values from least to greatest
```

Comment:
I would add the way we will ensure it sorts from least to greatest. Looks great!!! 💯